srcpkgs/zsh-fast-syntax-highlighting/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'zsh-fast-syntax-highlighting'
+pkgname=zsh-fast-syntax-highlighting
+version=1.55
+revision=1
+wrksrc="fast-syntax-highlighting-${version}"
+depends="zsh"
+short_desc="Feature-rich syntax highlighting for ZSH"
+maintainer="akhil <akhil.j@outlook.in>"
+license="BSD-3-Clause"
+homepage="https://github.com/zdharma-continuum/fast-syntax-highlighting"
+changelog="https://raw.githubusercontent.com/zdharma-continuum/fast-syntax-highlighting/master/changelog.md"
```

Comment:
404

Reply:
a changelog does exist in the repo, you just had the wrong url...

Reply:
removed it

Reply:
yes, add it back with the correct url...

Reply:
added